shared/constants/permissions.ts

```diff
@@ -31,8 +31,43 @@ export const RestrictedMethods = Object.freeze({
 export const ConnectionPermission = Object.freeze({
   connection_permission: 'connection_permission',
 });
+
+// This configuration specifies permission weight thresholds used to determine which
+// permissions to show or hide on certain Snap-related flows (Install, Update, etc.)
+export const PermissionWeightThreshold = Object.freeze({
+  snapInstall: 3 as const,
+  snapUpdateApprovedPermissions: 2 as const,
+});
 ///: END:ONLY_INCLUDE_IF
 
+export const PermissionWeight = Object.freeze({
+  eth_accounts: 3,
+  permittedChains: 3,
+  snap_dialog: 4,
+  snap_notify: 4,
+  snap_getBip32PublicKey: 3,
+  snap_getBip32Entropy: 1,
+  snap_getBip44Entropy: 1,
+  snap_getEntropy: 3,
+  snap_manageState: 4,
+  snap_getLocale: 4,
+  wallet_snap: 4,
+  endowment_networkAccess: 3,
+  endowment_webassembly: 4,
+  endowment_transactionInsight: 4,
+  endowment_cronjob: 4,
+  endowment_ethereumProvider: 4,
+  endowment_rpc: 4,
+  endowment_lifecycleHooks: 4,
+  endowment_pageHome: 4,
+  snap_manageAccounts: 2,
```

Comment:
Reconsidering this being a `2`. I don't think it needs to have a modal

Reply:
I have no strong opinion on this. This is why I had to update the e2e tests. Let see with other people and let me know if there is decision to change this. Also, we should update it in notion first, since it's source of truth for the weights.

Reply:
This is update to `3` now according to the Notion document.